src/System.Private.Reflection.Metadata.Ecma335/src/System.Private.Reflection.Metadata.Ecma335.csproj

```diff
@@ -1,253 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is this assembly being removed?

Reply:
`System.Private.Reflection.Metadata.Ecma335` was for .NET Native. (In case we need it for CoreRT experiments, we will add this in the CoreRT experimental branch.)